app/src/main/java/fr/racomach/zigweelo/model/User.kt

```diff
@@ -0,0 +1,6 @@
+package fr.racomach.zigweelo.model
+
+data class User(
+    val name: String?,
+    val authenticationToken: String
+)
```

Comment:
I should activate the option in IntelliJ to add a new line automatically at the end of the file